store/src/main/java/com/denis/store/utility/populator/Populator.java

```diff
@@ -0,0 +1,10 @@
+package com.denis.store.utility.populator;
+
+import com.denis.domain.Category;
+
+import java.util.List;
+
+public interface Populator {
+
+    List<Category> getRandomCategory();
```

Comment:
not very suitable method name. it is better to name is getCategoryList, or getCategories, or...